src/test/java/LocatorsTests.java

```diff
@@ -0,0 +1,74 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LocatorsTests {
+
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setup(){
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+
+    @BeforeMethod
+    public void input(){
+        driver.get("https://www.saucedemo.com/");
+    }
+
+    @AfterMethod
+    public void quit(){
+        driver.quit();
+    }
+
+    @Test
```

Comment:
Were you able to run these two tests at once? I think not, because the driver quits after the 1st test and second will fail. So it's better to quit/close the driver after all tests are done (@AfterClass)